packages/manager/apps/pci-kubernetes/src/pages/new/steps/PlanStep.component.tsx

```diff
@@ -72,12 +71,13 @@ const PlanTile = ({
   step,
   type,
 }: {
-  onSubmit: (plan: TClusterCreationForm['plan']) => void;
+  onSubmit: (plan: TClusterPlan) => void;
   step: StepState;
   type: DeploymentMode;
 }) => {
-  const [selected, setSelected] = useState<TClusterCreationForm['plan']>(
-    isMonoDeploymentZone(type) ? 'standard' : 'premium',
+  // When both free & standard plans will be available for both deployment zones, update default value with "free"
```

Comment:
I'll suggest to prefix the comment with `TODO` to make the future change more explicit :)